README.md

```diff
@@ -20,8 +20,8 @@ mvn clean install
 
 ## Release
 
-```bash
-mvn release:prepare release:perform -Prelease
-```
+To release a new version of Gizmo, follow these steps:
+
+https://github.com/smallrye/smallrye/wiki/Release-Process#releasing
```

Comment:
Could we maybe have a page in https://github.com/quarkusio/quarkus-release/wiki ?

I wouldn't tie us to SmallRye as it could evolve separately.

Not blocking, we can do it afterwards.